magento-engcom/import-export-improvements
# MAGETWO-70390 selection can change qty

Provides support for `selection_can_change_qty` during import and export

### Description
Adds bundle selection properties for use during import and export.

### Fixed Issues (if relevant)

1. magento/magento2#9342 (magento-engcom/import-export-improvements#41 )

### Manual testing scenarios
1. Create Simple products to assign to bundle.
2. Create Bundled products and assign simples as options.
3. Export via Import/Export. {{bundle_values}} column should have a property for {{can_change_qty}} which matches the value in for the option in the admin.
4. Change the value of {{can_change_qty}} and import using the Add/Replace method. Bundle option should change the "User Defined" checkbox in the admin.

### Contribution checklist
